src/pkg/clouds/azure/aca/cert.go

```
@@ -146,38 +168,64 @@ func findContainerAppByService(ctx context.Context, client *armappcontainers.Con
+// Azure yet, so a caller can add just this record ahead of a DNS cutover and
+// have the hostname registered/verified in advance.
+func waitForAsuidTXT(ctx context.Context, hostname, expectedTxt string, deadline time.Time, resolverAt func(string) dns.Resolver) error {
+	asuid := "asuid." + hostname
+	promptShown := false
+
+	for {
+		if txtOK, _ := dns.LookupTXTContains(ctx, asuid, expectedTxt, resolverAt("")); txtOK {
+			return nil
+		}
+		if !promptShown {
+			term.Printf("Configure DNS record for %s:\n", hostname)
```

golangci-lint (2.12.2)

[error] Error return value of `term.Printf` is not checked

(errcheck)